devolving_music/lib/Song_Scores.py

```diff
@@ -0,0 +1,39 @@
+from devolving_music.lib.elo_scoring import elo_rating
+from devolving_music.models.song_submission import SongSubmission
+from collections.abc import Iterable
```

Comment:
```suggestion
from typing import Iterable
```

idk what `collections.abc` is, but the `Iterable` typehint comes from the `typing` standard library package.